src/library_manager.py

```diff
@@ -77,22 +77,25 @@
 def get_storage_group(dict):
     sg = dict.get('storage_group', None)
     if not sg:
-	vf = dict.get('volume_family', None)
-	if vf:
-	    sg = volume_family.extract_storage_group(vf)
+        vf = dict.get('volume_family', None)
```

Comment:
Are those tabs ? I believe they need to change to white spaces.

Reply:
All of these instances look like white spaces to me in vim and pycharm. I'm not sure if it is an editor quirk but deleting and replacing with spaces gives me no change. Can you double check?

Thanks for the close look

Reply:
OK, let's assume these are not tabs. Not that super important